src/layouts/Settings/GeneralSettings.tsx

```diff
@@ -89,13 +89,16 @@ export const GeneralSettings = ({
             />
           </InputGroup>
           <FormErrorMessage>
-            The web domain you have entered is not valid.
+            {errors.url?.type === "required" &&
+              "This field cannot be left blank"}
+            {errors.url?.type === "pattern" &&
+              "The web domain you have entered is not valid"}
```

Comment:
i think we'd have to specify the type on `useFormState` like so: 

```ts
const { errors } = useFormState<SiteInfo>()
```

this allows the type of errors to be specified - if we hover over it rgiht now, it's typed as `any`